pubspec.yaml

```diff
@@ -6,59 +6,37 @@ authors:
   - Workiva UI Platform Chapter <uip@workiva.com>
   - Workiva App Frameworks Team <appframeworks@workiva.com>
 environment:
-  sdk: '>=2.0.0-dev.0 <=2.0.0-dev.49.0'
+  sdk: '>=2.1.0 <3.0.0'
```

Comment:
I thought these changes were slated for Dart 1 and Dart 2 SDK compat? Shouldn't the lower bound remain `>=1.24.3` in that case? Or am I misunderstanding the milesone?

Reply:
Transformers can't exist in a Dart 2 repo b/c `barback` doesn't have any Dart 2 compatible versions.

Reply:
mmm... we should probably change the milestone description then, IMO.  It says pretty clearly that we're targeting dart 1 and dart 2 support, which sounds like it isn't possible.

Reply:
We will still _effectively_ be supporting both Dart 1 and Dart 2, but the method for doing so will be the dual release lines (eg. `2.0.0-dart1` and `v2.0.0-dart2`).

Reply:
Yep, except it's `+`, not `-`, right? Also I believe that the plan is to have `2.0.0` and `2.0.0+dart1` (i.e. Dart 2 is the default)

Reply:
How will the dual release lines work if the SDK lower bound (and other dependency ranges) are Dart 2 only @evanweible-wf ?